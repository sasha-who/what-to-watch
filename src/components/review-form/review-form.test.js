import React from "react";
import renderer from "react-test-renderer";
import ReviewForm from "../review-form/review-form.jsx";

it(`ReviewForm should render correctly`, () => {
  const tree = renderer
    .create(
        <ReviewForm />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
