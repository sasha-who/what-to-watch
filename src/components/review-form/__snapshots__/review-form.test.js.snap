// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewForm should render correctly 1`] = `
<div
  className="add-review"
>
  <form
    action="#"
    className="add-review__form"
  >
    <div
      className="rating"
    >
      <div
        className="rating__stars"
      >
        <input
          className="rating__input"
          defaultChecked={false}
          defaultValue={0}
          id="star-0"
          name="rating"
          type="radio"
        />
        <label
          className="rating__label"
          htmlFor="star-0"
        >
          Rating 
          0
        </label>
        <input
          className="rating__input"
          defaultChecked={false}
          defaultValue={1}
          id="star-1"
          name="rating"
          type="radio"
        />
        <label
          className="rating__label"
          htmlFor="star-1"
        >
          Rating 
          1
        </label>
        <input
          className="rating__input"
          defaultChecked={true}
          defaultValue={2}
          id="star-2"
          name="rating"
          type="radio"
        />
        <label
          className="rating__label"
          htmlFor="star-2"
        >
          Rating 
          2
        </label>
        <input
          className="rating__input"
          defaultChecked={false}
          defaultValue={3}
          id="star-3"
          name="rating"
          type="radio"
        />
        <label
          className="rating__label"
          htmlFor="star-3"
        >
          Rating 
          3
        </label>
        <input
          className="rating__input"
          defaultChecked={false}
          defaultValue={4}
          id="star-4"
          name="rating"
          type="radio"
        />
        <label
          className="rating__label"
          htmlFor="star-4"
        >
          Rating 
          4
        </label>
      </div>
    </div>
    <div
      className="add-review__text"
    >
      <textarea
        className="add-review__textarea"
        id="review-text"
        name="review-text"
        placeholder="Review text"
      />
      <div
        className="add-review__submit"
      >
        <button
          className="add-review__btn"
          type="submit"
        >
          Post
        </button>
      </div>
    </div>
  </form>
</div>
`;
